test/Microsoft.AspNetCore.Mvc.FunctionalTests/compiler/resources/BasicWebSite.Area1.RemoteAttribute_Home.Create.html

```diff
@@ -83,10 +91,10 @@
 
     </div>
 
-    <script src="/lib/jquery/jquery.js"></script>
+    <script src="/javascript/jquery-2.1.4.min.js"></script>
```

Comment:
Yeah we don't want to check them in. Reference them from the CDN. And make the PR 10,000 lines smaller 😄